### ISSUE TITLE
Unsupported kind of type: ConstantArray

### ISSUE THREAD

**Issue by alexchandel** [rust-qt/cpp_to_rust#16]
Dealing with C++ code that uses networking on OSX invariably includes headers like `sys/_pthread/_pthread_types.h` which are full of structs with constant array members. C++ objects with fields that are (eventually) composed of these structs fail to parse.

Rust supports fixed-length arrays, so this should be easy to add.

**Comment by Riateche** [comment]
Yes, fixed size arrays can be supported. It was not a top priority because they are seldom used in API, but it should be fairly easy.

Note that information about class fields is currently not used. Failing to parse a field will not have any effect. The class and its methods should still be available.
